File: system_template/test/1_no_namespace/data/system_template/wd_test/_map.py
[
    {
        "source": "subpath/wd_test_json.behavior.json",
        "target": AUTO_FLAT,
        "json_template": True,
        "scope": {
            "os": os,
            "mappy_wd": os.getcwd()
        }
    },
    {
        "source": "subpath/wd_test_py.behavior.py",
        "target": AUTO_FLAT,
        "scope": {
            "os": os,
            "mappy_wd": os.getcwd()
        }
    },
    {
        "source": "subpath/wd_test.mcfunction",
        "target": AUTO_FLAT,
        "scope": {
            "os": os,
            "mappy_wd": os.getcwd()
        }
    }
]

File: modular_mc/test/packs/data/modular_mc/access/shared/_map.ts
import _resource_map from "../../_shared/_resource_map.ts";
const { sharedBehavior } = _resource_map;

export const MAP = [
    {
        source: sharedBehavior,
        target: ":autoFlat"
    },
    {
        source: sharedBehavior,
        target: {
            path: ":autoFlat",
            subpath: "subpath",
            name: ":auto"
        }
    }
]